testsuite/testcases/src/load_vs_perf_benchmark.rs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::NetworkLoadTest;
+use aptos_logger::info;
+use forge::{EmitJobMode, NetworkContext, NetworkTest, Result, Test, TxnStats};
+use rand::SeedableRng;
+use std::time::Duration;
+use tokio::runtime::Runtime;
+
+pub struct SingleRunStats {
+    tps: usize,
+    stats: TxnStats,
+    ledger_transactions: u64,
+    actual_duration: Duration,
+}
+
+pub struct LoadVsPerfBenchmark {
+    pub test: &'static dyn NetworkLoadTest,
+    pub tps: &'static [usize],
+}
+
+impl Test for LoadVsPerfBenchmark {
+    fn name(&self) -> &'static str {
+        "continuous progress test"
+    }
+}
+
+impl LoadVsPerfBenchmark {
+    fn evaluate_single(
+        &self,
+        ctx: &mut NetworkContext<'_>,
+        tps: usize,
+        duration: Duration,
+    ) -> Result<SingleRunStats> {
+        let rng = SeedableRng::from_rng(ctx.core().rng())?;
+        let emit_job_request = ctx.emit_job.clone().mode(EmitJobMode::ConstTps { tps });
+        let (stats, actual_duration, ledger_transactions) = self.test.network_load_test(
+            ctx.swarm(),
+            emit_job_request,
+            duration,
+            // add larger warmup, as we are exceeding the max load,
+            // and for that it takes more time to fill mempool.
+            0.2,
+            0.05,
+            rng,
+        )?;
+
+        Ok(SingleRunStats {
+            tps,
+            stats,
+            ledger_transactions,
+            actual_duration,
+        })
+    }
+}
+
+impl NetworkTest for LoadVsPerfBenchmark {
+    fn run<'t>(&self, ctx: &mut NetworkContext<'t>) -> Result<()> {
```

Comment:
Do you want to run this test in continuous mode or is it just for adhoc testing?

Reply:
At the moment adhoc.

If I think of success criteria for such a great,  we can make it run on continuous too.